modules/ROOT/pages/index.adoc

```diff
@@ -13,25 +14,28 @@ image::application-network.png[Application Network]
+
 
-From Anypoint Visualizer you can customize the view arrange applications in layers, inspect their operational monitoring information, and customize the labels displayed for nodes and connections. See xref:customize-your-canvas.adoc[Customize the Canvas] for more information.
+Understanding these dependencies has powerful implications — especially when you have the ability to overlay additional data about these apps onto the network diagram. The power of Visualizer, and the visualizations tailored for Architecture, Troubleshooting and Policies, is understanding this data in the context of your application network. From Anypoint Visualizer, you can inspect performance metrics or policy information, define views, arrange applications in layers, and customize the labels displayed for nodes and connections. See xref:customize-your-canvas.adoc[Customize the Canvas] for more information.
 
 == Use Cases
 
-The following use cases provide examples of how you can use Anypoint Visualizer.
+The following use cases provide examples of how you can use Anypoint Visualizer and its various visualizations.
+
+* Discussing Current Design State: Enterprise architect Kara has developed a plan for how the architecture she’s responsible for will accommodate a new product. She needs to capture its current state so she can plan the transition and get buy-in from stakeholders on her proposal. She can use the Architecture Visualization to assess the structure of her network and export it to share with her audience.
 
-* Discussing Current Design State: Enterprise architect Kara has developed a plan for how the architecture she’s responsible for will accommodate a new product. She needs to capture its current state so she can plan the transition and get buy-in from stakeholders on her proposal.
+* Implementing API-Led Connectivity: Site reliability engineer Tom is tired of maintaining several separate connections to Salesforce. He wants to identify redundant point-to-point connections so he can come up with requirements for a new process API. Tom can view the network topology using the Architecture Visualization to see if there are redundant connections.
 
-* Implementing API-Led Connectivity: Site reliability engineer Tom is tired of maintaining several separate connections to Salesforce. He wants to identify redundant point-to-point connections so he can come up with requirements for a new process API.
+* Exploring the Application Network: Anypoint Visualizer provides a real-time view of the application network. Architects and product owners can see how the application network evolves, explore the application network to discover and plan how to unlock data and break down siloed systems, while knowing that the visualization is always up-to-date. They can use the Architecture Visualization to plan how to connect systems more efficiently.
```

Comment:
```suggestion
* Exploring the Application Network
Anypoint Visualizer provides a real-time view of the application network. Architects and product owners can see how the application network evolves, explore the application network to discover and plan how to unlock data, and break down siloed systems, all while knowing that the visualization is always up-to-date. They can use the Architecture Visualization to plan how to connect systems more efficiently.
```

Reply:
I guessed about where to add a comma above. It did not make sense to me without it, nor did Grammarly like it.